src/test/scala/org/broadinstitute/dsde/firecloud/service/ApiServiceSpec.scala

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.broadinstitute.dsde.firecloud.Application
+import org.broadinstitute.dsde.firecloud.dataaccess.{MockAgoraDAO, MockRawlsDAO, MockSearchDAO, MockThurloeDAO}
+import org.broadinstitute.dsde.firecloud.mock.MockGoogleServicesDAO
+import org.broadinstitute.dsde.firecloud.webservice.NihApiService
+import org.scalatest.{FlatSpec, Matchers}
+import spray.httpx.SprayJsonSupport
+import spray.routing.HttpService
+import spray.testkit.ScalatestRouteTest
+
+import scala.concurrent.duration._
+
+/**
+  * Created by mbemis on 3/1/17.
+  */
+
+// common trait to be inherited by API service tests
```

Comment:
i've only done this for NihApiServiceSpec.

this pattern was stolen from rawls and should nicely address GAWB-1477